Added French localization

le-jun opened the pull request:
Translation is limited to the app description. Is there a way to translate the menu’s strings ?

le-jun committed:
French localization

eoineoineoin committed:
Add template French app translations file

eoineoineoin commented:
Hey, thanks for that!

It should be possible to translate the app, too. I made a branch based on your pull request with a template file in app/src/main/res/values-fr/strings.xml - editing all of those strings should get everything in French. The branch is https://github.com/eoineoineoin/ecwt/tree/french if you'd like to tackle that?

le-jun committed:
French localization

Translated multiple strings in french, except for what seemed to be variables (lines 23 to 29 and 34). Also, the last string (line 46) isn’t hard line breaked, feel free to correct it.

le-jun committed:
Merge pull request #1 from le-jun/patch-1

French localization

le-jun committed:
Merge pull request #2 from le-jun/patch-1

French localization

le-jun committed:
Merge pull request #3 from le-jun/french

Merge pull request #1 from le-jun/patch-1

le-jun commented:
Aaand it’s done!
